client/public/index.html

```diff
@@ -27,7 +27,7 @@
     </noscript>
     <div id="root"></div>
     <script type="text/javascript" charset="utf-8">
-      window.REDUX_STATE = "__SERVER_REDUX_STATE__";
+      window.REDUX_STATE = {};
```

Comment:
I add to change this to get rid of an error on `http://localhost:3000` (webpack dev server) when there's no SSR.